src/components/Nav/Nav.js

```diff
@@ -1,8 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
+import NavMore from './NavMore';
 import './Nav.scss';
 
 const Nav = () => {
-  return <div>Nav 입니다.</div>;
+  const [isActive, setIsActive] = useState(false);
+
+  return (
+    <div className="body">
+      <div className="wrapper">
+        <div className="navContainer">
+          <div
+            className="navLeft"
+            onMouseOver={() => {
+              setIsActive(true);
+            }}
+          >
+            <div className="insenseImg" />
+            <ul className="category">
+              <li>선물</li>
+              <li>인센스</li>
+              <li>홀더</li>
+              <li>브랜드</li>
+            </ul>
+          </div>
+          <div className="navRight">
+            <ul className="cartLogin">
+              <li>장바구니</li>
+              <li>로그인</li>
+            </ul>
+          </div>
+          {isActive ? <NavMore setIsActive={setIsActive} /> : null}
```

Comment:
```suggestion
          {isActive && <NavMore setIsActive={setIsActive} /> }
```
[논리연산자](https://ko.javascript.info/logical-operators)의 또 다른 기능인 단축평가로 줄일 수 있습니다.
이 문서를 참고 해 보세요